docs_src/general/ServicePorts.md

```diff
@@ -5,15 +5,19 @@ The following tables (organized by type of service) capture the default service
     |Services Name|	Port Definition|
     |---|---|
     |core-data|	59880|
-    |ZMQ - to be deprecated in a future release|5563|
     |core-metadata	|59881|
     |core-command	|59882|
+    |redis|6379|
+    |consul|8500|
+    |ZMQ - deprecated|5563|
 === "Supporting"
     |Services Name|	Port Definition|
     |---|---|
     |support-notifications	|59860|
     |support-scheduler|	59861|
-=== "Application & Analytics"
+    |rules engine / eKuiper|59720|
```

Comment:
eKuiper is part of Application & Analytics

Reply:
According to the architecture, it is under supporting services: https://docs.edgexfoundry.org/2.2/#supporting-services-layer

 The working group is Application & Analytics.

Reply:
Yea, the architecture diagram is out dated. I'm ok with the change to stay consistent with the diagram and adjust when/if the diagram is updated.